create-ocp-project/schemas/create-ocp-project__main-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "classpath:/schemas/create-ocp-project-schema.json",
+  "title": "Data input schema",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "inputs": {
+      "title": "Inputs",
+      "type": "object",
+      "properties": {
+        "operationsProjectKey": {
+          "title": "Operations Project Key",
+          "description": "JIRA project key for operations",
+          "type": "string",
+          "default": "OP"
+        },
+        "auditProjectKey": {
+          "title": "Audit Project Key",
+          "description": "JIRA project key for auditing",
+          "type": "string",
+          "default": "AUD"
+        },
+        "projectName": {
+          "title": "Project Name",
+          "description": "OCP project to create",
+          "type": "string"
```

Comment:
we should have add pattern to validate the name as https://github.com/parodos-dev/serverless-workflows/blob/main/request-vm-cnv/schemas/request-vm-cnv__main-schema.json#L21

Reply:
here you go: https://github.com/parodos-dev/serverless-workflows/pull/243